projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+    "item.doggytalents.soy_beans_dried": "干大豆",
+    "item.doggytalents.soy_pods_dried": "干豆荚",
+    "item.doggytalents.soy_pods_dried.description": "右键剥开豆荚",
+    "item.doggytalents.edamame": "毛豆",
+    "item.doggytalents.edamame.description": "又名盐水煮嫩豆。你和狗狗都能吃！\n但记得先剥壳！\n右键剥开豆荚",
+    "item.doggytalents.edamame_unpodded": "去壳毛豆",
+    "item.doggytalents.edamame_unpodded.description": "我开动了！",
+    "item.doggytalents.miso_paste": "味噌酱",
+    "item.doggytalents.miso_paste.description": "咸香！适合做味噌汤！",
+    "item.doggytalents.miso_soup": "味噌汤",
+    "item.doggytalents.miso_soup.description": "我开动了！",
+    "item.doggytalents.soy_milk": "豆浆",
+    "item.doggytalents.tofu": "豆腐",
+    "item.doggytalents.tofu.description": "素食主义者的最爱！",
+    "item.doggytalents.aburaage": "油炸豆腐",
+    "item.doggytalents.natto": "纳豆",
+    "item.doggytalents.natto.description": "黏糊糊的发酵豆子！",   
+    "item.doggytalents.aburaage.description": "也称为炸豆腐。用于制作稻荷寿司",
+    "item.doggytalents.onsen_tamago": "温泉蛋",
+    "item.doggytalents.onsen_tamago.description": "被称为温泉蛋。\n将鸡蛋丢入下方有燃烧方块（火、营火、岩浆块等）的炼药锅即可烹煮",
+    "item.doggytalents.gyudon": "牛丼",
+    "item.doggytalents.gyudon.description": "牛肉盖饭——食其家风味！",
+    "item.doggytalents.oyakodon": "亲子丼",
+    "item.doggytalents.oyakodon.description": "鸡肉鸡蛋盖饭——别深究名字含义！",
+    "item.doggytalents.natto_rice": "纳豆拌饭",
+    "item.doggytalents.natto_rice.description": "享用纳豆的最佳方式！", 
+    "item.doggytalents.sake": "清酒瓶",
+    "item.doggytalents.sake.description": "理性饮酒！未成年狗狗禁止饮用！",
+
+    "item.doggytalents.scent_treat": "气味零食",
+    "item.doggytalents.scent_treat.description": "右键方块采集气味样本。右键装水炼药锅清除气味",
+    "item.doggytalents.scent_treat.scented_block": "已采集方块样本：",
+    "item.doggytalents.drool_scent_treat": "沾满唾液的气味零食",
+
+    "item.doggytalents.feathered_mantle": "羽饰披风",
+
+    "item.doggytalents.grand_piano_black_item": "三角钢琴-黑色（实验性）",
+    "item.doggytalents.grand_piano_white_item": "三角钢琴-白色（实验性）",
+    "item.doggytalents.upright_piano_black_item": "立式钢琴-黑色（实验性）",
+    "item.doggytalents.upright_piano_brown_item": "立式钢琴-棕色（实验性）",
+    "items.doggytalents.piano_item_common.description": "手持火把潜行+右键调整方向",
+    "item.doggytalents.dog_plushie_toy_item": "狗狗毛绒玩具",
+    "item.doggytalents.dog_plushie_toy_item.description": "手持火把潜行+右键调整方向。右键狗狗可复制其变种和项圈样式",
+    "item.doggytalents.cape_coloured.tooltip": "颜色：%s",
+    "item.doggytalents.wool_collar.tooltip": "颜色：%s",
+    "item.doggytalents.creative_collar.tooltip": "提供无限技能点数",
+    "item.doggytalents.creative_radar.tooltip": "可追踪所有驯养的狗狗",
+    "item.doggytalents.treat_bag.contents": "内含：",
+    "item.doggytalents.treat_bag.contents.more": "另有%d件物品......",
+    "item.doggytalents.treat_bag.help": "右键点击可以远程投掷食物！潜行+右键打开零食袋的物品栏",
+    "item.doggytalents.conducting_bone.fulfilled.tp_self": "已将%s传送到主人身边！",
+    "item.doggytalents.conducting_bone.fulfilled.tp_bed": "%s正躺在%s的床上",
+    "item.doggytalents.conducting_bone.rejected": "指挥失败！错误代码：%s",
+    "item.doggytalents.radar.status" : "狗狗追踪器 - \"%s\"",
```

```suggestion
    "item.doggytalents.radar.status" : "狗狗追踪器 - “%s”",
```
理论上在不影响显示效果的情况下，应当使用汉语引号